mlabs/src/Mlabs/EfficientNFT/Token.hs

```diff
@@ -154,3 +154,27 @@ policy oref authorPkh royalty platformConfig contentHash =
       `PlutusTx.applyCode` PlutusTx.liftCode royalty
       `PlutusTx.applyCode` PlutusTx.liftCode platformConfig
       `PlutusTx.applyCode` PlutusTx.liftCode contentHash
+
+-- Error messages
+errUtxoNotConsumed :: BuiltinString
```

Comment:
I don't think that hiding the error messages behind a variable is going to be helpful